website/meta/universe.json

```
@@ -1,5 +1,33 @@
 {
     "resources": [
+        {
+            "id": "latincy",
+            "title": "LatinCy",
+            "thumb": "https://raw.githubusercontent.com/diyclassics/la_core_web_lg/main/latincy-logo.png",
+            "slogan": "Synthetic trained spaCy pipelines for Latin NLP",
+            "description": "Set of trained general purpose Latin-language 'core' pipelines for use with spaCy. The models are trained on a large amount of available Latin data, including all five of the Latin Universal Dependency treebanks, which have been preprocessed to be compatible with each other.",
+            "url": "https://huggingface.co/latincy",
+            "code_example": [
```

I'm not sure whether this is exactly how you'd want to specify this, but I think it would be helpful to have a comment in the code the code example about how to install the pipeline?

```suggestion
            "code_example": [
                "# pip install https://huggingface.co/latincy/la_core_web_lg/resolve/main/la_core_web_lg-any-py3-none-any.whl",
```

---

Good suggestion—thanks!